pkg/kubelet/rkt/version.go

```diff
@@ -54,22 +54,13 @@ func (r rktVersion) Compare(other string) (int, error) {
 	return 0, nil
 }
 
-// checkVersion tests whether the rkt/systemd/rkt-api-service that meet the version requirement.
-// If all version requirements are met, it returns nil.
-func (r *Runtime) checkVersion(minimumRktBinVersion, recommendedRktBinVersion, minimumAppcVersion, minimumRktApiVersion, minimumSystemdVersion string) error {
-	// Check systemd version.
+func (r *Runtime) getVersions() error {
```

Comment:
Hm...the r.*Version fields are written in the function and called periodically through Status(), but the fields can also be read by Version() or APIVersion() functions. Seems like a data race.